smartsim/_core/mli/infrastructure/environment_loader.py

```diff
@@ -72,7 +73,8 @@ def get_backbone(self) -> t.Optional[FeatureStore]:
         an environment variable. The backbone is a standalone, system-created
```

Comment:
I know you did not write this, but since we know (and mention in `:returns:`) that the env variable is `_SMARTSIM_INFRA_BACKBONE`, should we write it here?